extensions/sql-migration/src/service/contracts.ts

```diff
@@ -548,3 +554,25 @@ export interface TdeMigrateProgressParams {
 	success: boolean;
 	message: string;
 }
+
+export interface SchemaMigrationParams {
+	sourceConnectionString: string;
+	sourceDatabaseName: string;
+	targetConnectionString: string;
+	targetDatabaseName: string;
+}
+
+export interface SchemaMigrationResult {
+	sourceDatabaseName: string;
+	targetDatabasename: string;
+	status: string;
+	errorMessage: string;
+}
+
+export namespace SchemaMigrationRequest {
```

Comment:
assuming we're not gonna merge this until STS with the NuGet is built and released - we need to update the version of the migration service being used here: https://github.com/microsoft/azuredatastudio/blob/main/extensions/sql-migration/config.json

Reply:
I would also vbump the version of the extension as well

Reply:
yeah I still need to push the sts changes as well